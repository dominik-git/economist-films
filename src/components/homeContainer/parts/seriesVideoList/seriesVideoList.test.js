import React from 'react';
import renderer from 'react-test-renderer';
import SeriesVideoList from './seriesVideoList';

const data = [
  { title: 'xxx',
    id: '5',
    items:[
      {
        thumbnail:{
          url:"https://cdn.twivel.io/uploads/economist/episode/thumbnail/142/875X480.jpg"
        }
      }
    ]
   }
];

test('SeriesVideoList renders correctly', () => {
  const SeriesVideoList = jest.fn((props) => <div>hello</div>);
  const tree : string = renderer.create(<SeriesVideoList shelves={data} />).toJSON();
  expect(tree).toMatchSnapshot();
});
