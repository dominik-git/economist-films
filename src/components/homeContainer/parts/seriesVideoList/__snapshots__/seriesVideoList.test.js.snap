exports[`test SeriesVideoList renders correctly 1`] = `
<div>
  hello
</div>
`;
